README.md

```diff
@@ -160,3 +160,11 @@ To download the binaries, run:
 curl -Lo kyma https://storage.googleapis.com/kyma-cli-stable/kyma-darwin # kyma-linux or kyma.exe
 chmod +x kyma
 ```
+
+### Post-release steps
+
+Once the CLI release is complete, the following steps need to be performed:
+
+1. Remove the existing `stable` tag and re-create it pointing to the latest commit on the `master` branch. Pushing this tag will trigger a postsubmit job that builds and pushes the latest stable binaries to a storage bucket. These are used by the continuous integration jobs.
```

Comment:
Please remove the `stable` step as we decided to do it independently of the release.